x-pack/qa/evil-tests/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/KerberosTestCase.java

```diff
@@ -59,6 +59,8 @@ public abstract class KerberosTestCase extends ESTestCase {
     /*
      * Arabic and other language have problems due to handling of generalized time in SimpleKdcServer. For more, look at
      * org.apache.kerby.asn1.type.Asn1GeneralizedTime#toBytes
+     *
+     * Note: several unsupported locales were added in CLDR. #109670 included these below.
```

Comment:
Git blame gives us this info too, but it feels like a nice time saver to have the context and PR ref ready.